### PR TITLE
Spanish updates

### LINKED ISSUE #3161: Spanish translation
**Description**

The spanish translation has a some definition problems:

Examples: 

- Room is defined as Habitación, and in the context of the word, it should be Sala since it's more broad than Habitación.

- When a user leaves it says: "<username> Izquierda", when it should be "<username> se fue" or "abandonó la sala". (it translated "left" literally).

- The button mute all, says "Silenciar todo", and it should say "Silenciar a todos". Meaning that you silence people, not media or everything that is making noise.

I posted a spanish translation in the discussion #3108  that was more precise for the understanding of a native spanish speaker from a neutral localization, since spanish can vary among countries, and it was done checking everything that we could find in the "en" language file.

### PR DESCRIPTION
**Pull request by keianhzo**
Closes #3161 Spanish updates

### CODE REVIEW

**Comment by keianhzo** [comment]
@elmau can you take a look at this and confirm that everything is ok? :) Gracias!

**Comment by elmau** [comment]
> 
> 
> @elmau can you take a look at this and confirm that everything is ok? :) Gracias!

I'm checking it right now

File: src/assets/locales/es.json [hunk 1]
```diff
@@ -1,169 +1,169 @@
...
-  "sign-in.hub": "Se requiere una cuenta para unirse a las salas.\n\nIntroduce tu correo electrónico para crear tu cuenta o iniciar sesión.",
-  "sign-in.auth-started": "¡Correo electrónico enviado a {email}!\n\nPara continuar, haz clic en el enlace del correo electrónico con tu teléfono, tableta o PC.\n\n¿No tienes correo electrónico? En ese caso quizá no puedas crear una cuenta.",
-  "sign-in.pin": "Debes iniciar sesión para anclar objetos",
+  "app-description": "Comparte una sala virtual con amigos.\nMira videos, juega con objetos 3D o simplemente pasa el rato.",
+  "app-tagline": "VR social privado en tu navegador web",
+  "auth.verified-title": "¡Correo verificado!",
+  "auth.verify-failed": "Imposible registrarse con este vínculo. Puede que ya haya sido usado o haya expirado.",
+  "auth.verified": "¡Tu correo ha sido verificado!\nAhora puedes cerrar esta pestaña y volver a %app-name%.",
+  "auth.spoke-verified": "¡Tu correo ha sido verificado!\nAhora puedes cerrar esta pestaña y volver a %editor-name%.",
+  "sign-in.prompt": "Inicia sesión para fijar objetos en salas.",
```
Comment (elmau) [inline src/assets/locales/es.json:13]: "Inicia sesión para fijar objetos en salas." should be "Inicia sesión para fijar objetos en las salas."

File: src/assets/locales/es.json [hunk 2]
```diff
@@ -1,169 +1,169 @@
...
-  "entry.leave-room": "Salir de la habitación",
+  "entry.room": "antesala",
+  "entry.enter-room-title": "Antesala",
+  "entry.enter-room": "Entrar a la sala",
+  "entry.leave-room": "Abandonar la sala",
   "entry.change-scene": "Elegir escena",
-  "entry.in-lobby-notice": "Está viendo esta habitación desde el vestíbulo.",
-  "entry.screen-prefix": "Entrar en ",
-  "entry.desktop-screen": "Pantalla",
+  "entry.in-lobby-notice": "Estás mirando esta sala desde la antesala.",
```
Comment (elmau) [inline src/assets/locales/es.json:83]: "Estás mirando esta sala desde la antesala." should be "Estás viendo esta sala desde la antesala."

File: src/assets/locales/es.json [hunk 3]
```diff
@@ -1,169 +1,169 @@
...
+  "entry.watch-from-lobby": "Mirar desde la antesala",
+  "entry.watch-from-lobby-subtitle": "Los demás no podrán verte ni escucharte",
+  "entry.cardboard": "Entrar con Google Cardboard",
   "entry.checkingForDeviceAvailability": "Comprobando la disponibilidad del dispositivo...",
-  "entry.daydream-prefix": "Entrar en ",
+  "entry.daydream-prefix": "Entrar con ",
   "entry.daydream-medium": "Daydream",
-  "entry.daydream-via-chrome": "Con Google Chrome",
-  "entry.invite-others": "invitar a otros",
+  "entry.daydream-via-chrome": "Usando Google Chrome",
```
Comment (elmau) [inline src/assets/locales/es.json:106]: "Usando Google Chrome" should be "Google Chrome"

File: src/assets/locales/es.json [hunk 4]
```diff
@@ -1,169 +1,169 @@
...
-  "profile.display_name.validation_warning": "Alfanuméricos y guiones. Al menos 3 caracteres, no más de 32",
+  "profile.display_name.validation_warning": "Caracteres alfanuméricos y guiones. Al menos 3 caracteres, máximo 32",
   "profile.header": "Nombre y avatar",
-  "profile.terms_of_use": "Condiciones de uso",
+  "profile.terms_of_use": "Términos de uso",
   "profile.privacy_notice": "Aviso de privacidad",
-  "profile.choose_avatar": "Explorar avatares",
-  "profile.tabs.legacy": "Predeterminado",
-  "profile.tabs.skinnable": "Skin personalizada",
+  "profile.choose_avatar": "Elegir avatares",
```
Comment (elmau) [inline src/assets/locales/es.json:122]: "Elegir avatares" should be "Elegir avatar"

File: src/assets/locales/es.json [hunk 5]
```diff
@@ -1,169 +1,169 @@
...
-  "media-browser.search-placeholder.videos": "Buscar videos...",
-  "media-browser.search-placeholder.images": "Buscar imágenes...",
-  "media-browser.search-placeholder.youtube": "Buscar videos de Youtube...",
-  "media-browser.search-placeholder.gifs": "Buscar GIF...",
-  "media-browser.search-placeholder.twitch": "Buscar emisiones de Twitch...",
-  "media-browser.search-placeholder.sketchfab": "Buscar modelos de Sketchfab...",
-  "media-browser.search-placeholder.poly": "Buscar modelos de Google Poly...",
+  "media-browser.search-placeholder.videos": "Buscar videos…",
+  "media-browser.search-placeholder.images": "Buscar imágenes…",
+  "media-browser.search-placeholder.youtube": "Buscar videos de YouTube…",
```
Comment (elmau) [inline src/assets/locales/es.json:134]: the "…" looks different from the other ". . ."

File: src/assets/locales/es.json [hunk 6]
```diff
@@ -1,169 +1,169 @@
...
-  "media-browser.empty.twitch": "No hay resultados. Intenta realizar una nueva búsqueda arriba.",
-  "media-browser.empty.favorites": "No tienes ningún favorito. Haz clic en ⭐ para agregarlo a tus favoritos",
+  "media-browser.empty.images": "No hay resultados. Prueba otra búsqueda arriba.",
+  "media-browser.empty.videos": "No hay resultados. Prueba otra búsqueda arriba.",
+  "media-browser.empty.youtube": "No hay resultados. Prueba otra búsqueda arriba.",
+  "media-browser.empty.gifs": "No hay resultados. Prueba otra búsqueda arriba.",
+  "media-browser.empty.sketchfab": "No hay resultados. Prueba otra búsqueda arriba.",
+  "media-browser.empty.poly": "No hay resultados. Prueba otra búsqueda arriba.",
+  "media-browser.empty.twitch": "No hay resultados. Prueba otra búsqueda arriba.",
+  "media-browser.empty.favorites": "No tienes ningún favorito. Haz clic en ⭐ para agregar a tus favoritos.",
```
Comment (elmau) [inline src/assets/locales/es.json:162]: "No tienes ningún favorito. Haz clic en ⭐ para agregar a tus favoritos." should be "No tienes ningún favorito. Haz clic en ⭐ para agregarlo a tus favoritos."

File: src/assets/locales/es.json [hunk 7]
```diff
@@ -173,288 +173,288 @@
...
-  "audio.subtitle-desktop": "Confirmar salida de altavoz HMD",
-  "audio.subtitle-mobile": "Se recomiendan auriculares",
-  "audio.enter-now": "Ingresar ahora",
-  "audio.hmd-mic-warning": "No se ha elegido un micrófono HMD",
-  "audio.grant-title": "Concede permisos de acceso al micrófono",
-  "audio.grant-subtitle": "Para ser escuchado por otros, concede permisos de acceso al micrófono",
-  "audio.granted-title": "Permisos de acceso al micrófono concedidos",
-  "audio.granted-subtitle": "Aún puedes silenciarte en el juego",
+  "media-browser.similar-to-facet": "Parecido a: \"{name}\"",
+  "audio.title": "Configuración de sonido",
```
Comment (elmau) [inline src/assets/locales/es.json:177]: "Configuración de sonido" should be "Configuración de audio"

File: src/assets/locales/es.json [hunk 8]
```diff
@@ -173,288 +173,288 @@
...
+  "preferences.closeButton": "Cerrar menú de preferencias",
   "preferences.selectCategory": "Seleccionar categoría",
-  "preferences.resetToDefault": "Restablecer preferencias a las predeterminadas",
-  "preferences.promptForRefresh": "Tus preferencias están guardadas, pero algunos de tus cambios no tendrán efecto hasta que actualices la página. ",
-  "preferences.refreshNow": "Actualizar ahora",
-  "preferences.category_audio": "Audio",
+  "preferences.resetToDefault": "Restaurar preferencias por defecto",
+  "preferences.promptForRefresh": "Tus preferencias están guardadas, pero algunos de los cambios no entrarán en vigencia hasta que refresques la página.",
+  "preferences.refreshNow": "Refrescar ahora",
+  "preferences.category_audio": "Sonido",
```
Comment (elmau) [inline src/assets/locales/es.json:286]: "Sonido" should be "Audio"

File: src/assets/locales/es.json [hunk 9]
```diff
@@ -173,288 +173,288 @@
...
-  "commands.audiomode": "(experimental) Alternar audio posicional.",
-  "preferences.closeButton": "Cerrar el menú de preferencias ",
+  "commands.rename": "Renombrar la sala.",
+  "commands.help": "Mostrar la ayuda.",
+  "commands.leave": "Desconectarse de la sala.",
+  "commands.duck": "El pato pasó bien el test. Cuac.",
+  "commands.capture": "Grabar un video de 15 segundos.",
+  "commands.debug": "Alternar el procesamiento de depuración de la física.",
+  "commands.vrstats": "Alternar estadísticas en VR.",
+  "commands.audiomode": "(experimental) Alternar sonido posicional.",
```
Comment (elmau) [inline src/assets/locales/es.json:280]: "(experimental) Alternar sonido posicional." should be "(experimental) Alternar audio posicional."

File: src/assets/locales/es.json [hunk 10]
```diff
@@ -173,288 +173,288 @@
...
+  "preferences.resetToDefault": "Restaurar preferencias por defecto",
+  "preferences.promptForRefresh": "Tus preferencias están guardadas, pero algunos de los cambios no entrarán en vigencia hasta que refresques la página.",
+  "preferences.refreshNow": "Refrescar ahora",
+  "preferences.category_audio": "Sonido",
   "preferences.category_controls": "Controles",
-  "preferences.category_misc": "Varios",
+  "preferences.category_misc": "Misceláneo",
   "preferences.category_movement": "Movimiento",
-  "preferences.category_touchscreen": "Pantalla táctil",
+  "preferences.category_touchscreen": "Pantallas táctiles",
```
Comment (elmau) [inline src/assets/locales/es.json:290]: "Pantallas táctiles" should be "Pantalla táctil"

File: src/assets/locales/es.json [hunk 11]
```diff
@@ -173,288 +173,288 @@
...
+  "preferences.enableOnScreenJoystickLeft": "Habilitar el controlador para caminar, a la izquierda de la pantalla",
+  "preferences.enableOnScreenJoystickRight": "Habilitar el controlador para girar, a la derecha de la pantalla",
   "preferences.invertTouchscreenCameraMove": "Invertir la dirección del movimiento de la cámara para pantallas táctiles",
-  "preferences.browserDefault": "Navegador predeterminado",
+  "preferences.browserDefault": "Navegador por defecto",
   "preferences.locale": "Idioma",
-  "preferences.onlyShowNametagsInFreeze": "Mostrar etiquetas de nombre solo mientras estás congelado",
-  "preferences.animateWaypointTransitions": "Animar transiciones de puntos de referencia",
+  "preferences.onlyShowNametagsInFreeze": "Solo mostrar las etiquetas de nombre en modo contextual",
+  "preferences.animateWaypointTransitions": "Animar transición de trayectoria",
```
Comment (elmau) [inline src/assets/locales/es.json:311]: "Animar transición de trayectoria" should be "Animar transiciones de puntos de referencia"

File: src/assets/locales/es.json [hunk 12]
```diff
@@ -173,288 +173,288 @@
...
-  "home.powered_by_link": "Cloud Hubs",
+  "home.room_create_button": "Crear una sala",
+  "home.create_name.validation_warning": "Nombre inválido. Debe tener entre 4 a 64 caracteres y símbolos limitados.",
+  "home.powered_by_prefix": "Hecho con la tecnología de",
+  "home.powered_by_link": "Hubs Cloud",
   "home.join_us": "Únete a la conversación",
-  "home.subscribe_to_mailing_list": "Suscríbete para recibir actualizaciones",
-  "home.have_code": "¿Tienes un código de habitación?",
-  "home.add_to_discord_1": "Agregar el",
+  "home.subscribe_to_mailing_list": "Suscríbete para actualizaciones",
```
Comment (elmau) [inline src/assets/locales/es.json:225]: "Suscríbete para actualizaciones" should be "Suscríbete para recibir actualizaciones"

File: src/assets/locales/es.json [hunk 13]
```diff
@@ -173,288 +173,288 @@
...
-  "tips.mobile.feedback": "🦆 ¡Quack! ¿Quieres ayudar a Hubs?",
-  "tips.mobile.feedback-link": "Dile al pato.",
-  "tips.desktop.video_share_mode": "Estás transmitiendo. Selecciona nuevamente para dejar de compartir videos.",
+  "tips.mobile.object_recenter_button-post": " para hacer que el objeto mire hacia ti.",
+  "tips.mobile.object_pin": "Fija un objeto para guardarlo dentro de la sala.",
+  "tips.mobile.video_share_failed": "No hay permiso otorgado.",
+  "tips.mobile.invite": "Usa el botón Compartir para compartir esta sala.",
+  "tips.mobile.feedback": "🦆 ¡Cuac! ¿Quieres ayudar a Hubs?",
+  "tips.mobile.feedback-link": "Cuéntale al pato.",
+  "tips.desktop.video_share_mode": "Estás haciendo una transmisión. Selecciona nuevamente para dejar de compartir el video.",
```
Comment (elmau) [inline src/assets/locales/es.json:363]: "Estás haciendo una transmisión. Selecciona nuevamente para dejar de compartir el video." should be "Estás transmitiendo. Selecciona nuevamente para dejar de compartir el video."

**Review by elmau — CHANGES_REQUESTED** [review]
I wrote some notes around the file with the corrected text, I don't know how to submit corrections here, so I resorted to that.

**Comment by keianhzo** [comment]
@elmau Updated, in case you want to take a look and confirm that everything looks correct.

**Review by johnshaughnessy — APPROVED** [review]
I don't speak Spanish well enough to comment on the translations and I trust your judgment. Just leaving this review in case I was blocking you from merging it.

**Comment by elmau** [comment]
Everything looks good with the changes. 👍 Will this be available soon?

**Review by misslivirose — APPROVED** [review]

**Pull request merged**